src/ut_kvp.c

```diff
@@ -110,11 +125,22 @@ ut_kvp_status_t ut_kvp_open(ut_kvp_instance_t *pInstance, char *fileName)
         return UT_KVP_STATUS_PARSING_ERROR;
     }
 
+    node = process_node(fy_document_root(pInternal->fy_handle), 0);
+    remove_include_keys(node);
+
+    if(node == NULL)
+    {
+         UT_LOG_ERROR("Unable to process node");
```

Comment:
did you test this case?

you've got an extra space before UT_

Reply:
No, I didn't as they all are static functions. Do I need to test these??